pkg/apis/servicecatalog/checksum/unversioned/checksum.go

```diff
@@ -38,6 +39,10 @@ func InstanceSpecChecksum(spec servicecatalog.InstanceSpec) string {
 	if spec.Parameters != nil {
 		specString += fmt.Sprintf("parameters:\n\n%v\n\n", string(spec.Parameters.Raw))
 	}
+	if spec.AlphaSecretParameters != nil {
+		// TODO: calc checksum of secret contents instead?
```

Comment:
>I would prefer the secret contents checksum to be addressed along with secret update support in a separate PR

I agree. I have missed the last few design meetings, but from what I understand, the group is planning to address secret updates in a few phases. This area should fall into that work.

That said, @nilebox can you add an issue for this specific work item, or perhaps mention it in an existing secret if appropriate?